Sources/CorePayments/AnalyticsEventData.swift

```diff
@@ -101,7 +102,7 @@ struct AnalyticsEventData: Encodable {
         self.correlationID = correlationID
     }
     
-    func encode(to encoder: Encoder) throws {
+    nonisolated func encode(to encoder: Encoder) throws {
         var topLevel = encoder.container(keyedBy: TopLevelKeys.self)
```

Comment:
This just gave warning in the during compile time but caused failure in Cocoapods test in the pipeline.

Reply:
Curious if you could add the Sendable protocol to this struct to resolve this issue? 

```struct AnalyticsEventData: Encodable, Sendable {```